LiteCore/Query/SQLitePredictionFunction.cc

```diff
@@ -108,6 +108,14 @@ namespace litecore {
         return (i1.count() == i2.count());
     }
 
+    static void writeResultNumeric(sqlite3_context* ctx, double num){
```

Comment:
I don't think this is necessary, since these results are unlikely to be integral.

Reply:
More than half of our assertions on this method use integral results.  Is it ok to change all of them to float instead (query distance functions)

Reply:
Sure.